### ISSUE TITLE
Build failing

### ISSUE THREAD

**Issue by lbischof** [antoiner77/caddy-ansible#21]
The build is failing. I don't have time at the moment to check what's wrong... 

```
TASK [caddy-ansible : Start Caddy service] *************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "start: Unknown job: caddy\n"}
```

**Comment by davidak** [comment]
@lbischof please provide a little more information.

What OS do you deploy to?
What Ansible and Python verion do you have on the controller?
How does your configuration look like (caddy part)?

**Comment by antoiner77** [comment]
Just take a look at travis-ci
https://travis-ci.org/antoiner77/caddy-ansible/builds/163937644

**Comment by lbischof** [comment]
This file would fix it, but it's not in the stable caddy release yet. https://github.com/mholt/caddy/blob/master/dist/init/linux-upstart/caddy.conf.ubuntu-12.04
I will make a pullrequest and merge when it's released

**Comment by lbischof** [comment]
Fixed by c433db8
